model/MenuModel.ts

```diff
@@ -24,8 +25,8 @@ class MenuModel {
                 menuItems: [String],
                 menuSections: Object,
                 menuDescription: String,
-                menuStartTime: Date,
-                menuEndTime: Date
+                menuStartTime: String,
+                menuEndTime: String
```

Comment:
```suggestion
                menuStartTime: string,
                menuEndTime: string
```

to be consistent with menuStart and menuEnd in other file